x-pack/legacy/plugins/uptime/common/constants/index_names.ts

```diff
@@ -5,5 +5,5 @@
  */
 
 export const INDEX_NAMES = {
-  HEARTBEAT: 'heartbeat*',
+  HEARTBEAT: 'heartbeat-7*',
```

Comment:
Hmmm, looks like I missed this one in the settings PR, will fix in a separate PR